habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/hierarchical_oracle_nav.yaml

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
let's rename this file to be "hierarchical_nn_oracle_skills.yaml"?
since all low-level skills are oracle (assuming that noop is the best approximation to oracle manipulation skills we have so far)?